src/osp/Resource/AssetImporter.h

```diff
@@ -157,6 +159,17 @@ PluginManager;
                                Magnum::UnsignedInt parentProtoIndex,
                                Magnum::UnsignedInt childGltfIndex);
 
+    static std::shared_ptr<spdlog::logger> get_logger() 
+    {
+        if (logger == nullptr) {
+            logger = spdlog::stdout_color_mt("assetimporter");
```

Comment:
don't do lazy initialization.

Require that the logger be passed in when the assertimporter is constructed, and assert that it is not null.

Reply:
AssetImporter is used basically fully statically, which is probably why he did it that way

Reply:
Then we should probably store the logger in entt for the asset importer, or initialize a global variable for the asset importer to pick up.